Composer/packages/client/src/pages/design/exportSkillModal/content/SelectProfile.tsx

```diff
@@ -55,6 +68,25 @@ const onRenderLabel = (props) => {
   );
 };
 
+const onRenderInvalidProfileWarning = (hasValidProfile, handleShowCreateProfileDialog) => {
+  return (
+    <div>
+      <span>{formatMessage('Publish profile is missing App ID and host name. ')}</span>
```

Comment:
shouldn't this be in a warning color? consult UX please

Reply:
Ask them about using something like:
https://developer.microsoft.com/en-us/fluentui#/controls/web/messagebar (warning)

Reply:
Fine, I will replace it.